allennlp/data/dataset_readers/sequence_tagging.py

```diff
@@ -23,18 +23,27 @@ class SequenceTaggingDatasetReader(DatasetReader):
 
     WORD###TAG [TAB] WORD###TAG [TAB] ..... \n
 
-    and converts it into a ``Dataset`` suitable for sequence tagging.
+    and converts it into a ``Dataset`` suitable for sequence tagging. You can also specify
+    alternative delimiters in the constructor.
 
     Parameters
     ----------
+    word_tag_delimiter: ``str``, optional (default=``"###"``)
+        The text that separates each WORD from its TAG.
+    word_word_delimiter: ``str``, optional (default=``"\t"``)
+        The text that separates each WORD-TAG pair from the next pair.
     token_indexers : ``Dict[str, TokenIndexer]``, optional (default=``{"tokens": SingleIdTokenIndexer()}``)
         We use this to define the input representation for the text.  See :class:`TokenIndexer`.
         Note that the `output` tags will always correspond to single token IDs based on how they
         are pre-tokenised in the data file.
     """
     def __init__(self,
+                 word_tag_delimiter: str = "###",
+                 word_word_delimiter: str = "\t",
                  token_indexers: Dict[str, TokenIndexer] = None) -> None:
         self._token_indexers = token_indexers or {"tokens": SingleIdTokenIndexer()}
+        self.word_tag_delimiter = word_tag_delimiter
```

Comment:
Preference for private variables here, unless you really want to make this part of the public API.